schema-engine/sql-introspection-tests/tests/views/postgresql.rs

```diff
@@ -103,6 +104,80 @@ async fn simple_view_from_one_table(api: &mut TestApi) -> TestResult {
     Ok(())
 }
 
+// PostgreSQL 16 only
+// the expect_view_definition is slightly different than for previous versions
+#[test_connector(tags(Postgres16), preview_features("views"))]
+async fn simple_view_from_one_table_postgres16(api: &mut TestApi) -> TestResult {
+    let setup = indoc! {r#"
+        CREATE TABLE "User" (
+            id SERIAL PRIMARY KEY,
+            first_name VARCHAR(255) NOT NULL,
+            last_name VARCHAR(255) NULL
+        );
+
+        CREATE VIEW "Schwuser" AS
+            SELECT id, first_name, last_name FROM "User";
+    "#};
+
+    api.raw_cmd(setup).await;
+
+    let expected = expect![[r#"
+        generator client {
+          provider        = "prisma-client-js"
+          previewFeatures = ["views"]
+        }
+
+        datasource db {
+          provider = "postgresql"
+          url      = "env(TEST_DATABASE_URL)"
+        }
+
+        model User {
+          id         Int     @id @default(autoincrement())
+          first_name String  @db.VarChar(255)
+          last_name  String? @db.VarChar(255)
+        }
+
+        /// The underlying view does not contain a valid unique identifier and can therefore currently not be handled by Prisma Client.
+        view Schwuser {
+          id         Int?
+          first_name String? @db.VarChar(255)
+          last_name  String? @db.VarChar(255)
+
+          @@ignore
+        }
+    "#]];
+
+    api.expect_datamodel(&expected).await;
+
+    // For Postgres <16 it looks like this:
+    // SELECT
+    //   "User".id,
+    //   "User".first_name,
+    //   "User".last_name
+    // FROM
+    //   "User";
+    let expected = expect![[r#"
+        SELECT
+          id,
+          first_name,
+          last_name
+        FROM
+          "User";"#]];
```

Comment:
Where does this view definition come from? Is this the direct result of a query in Introspection?

Reply:
It comes from introspection indeed.

Reply:
Deserves an issue to look into then when we work on `views` again.
(Postgres 16 exists, so not much else we can do about it)